### DISCUSSION TITLE
Rename `store` to `stores`

### DISCUSSION THREAD

**Discussion by posva** [nuxtrdev/nuxtr-vscode#12] [Ideas]
Hello! cool extension!
I think the default folder for stores should have an `s` as that's the recommended naming in pinia and it aligns with the fact that an application has multiple stores rather than a root store with modules (like in Vuex)

**Comment by adhamfarrag** [comment]
Means a lot coming from you Eduardo!

Fixed and will be pushed soon.

**Comment by adhamfarrag** [comment]
Actually, is there anything you'd like to have on this side? Saw your tweet about [Pinia Snipets](https://pinia.vuejs.org/cookbook/vscode-snippets.html#vscode-snippets). Is there anything you'd like to be offered out of the box either on this side or something else ?

**Reply by posva** [reply]
Feel free to add it and tweak it as you want. I don't have anything in particular in mind 😄
